tests/test_getId.py

```diff
@@ -57,7 +57,9 @@ def testId(self):
         bits = self.butler.get("deepCoaddId_bits", dataId, immediate=True)
         id = self.butler.get("deepCoaddId", dataId, immediate=True)
         self.assertEqual(bits, 37)
-        self.assertEqual(id, ((((1 * 8192) + 2) * 8192) + 3)*8 + 4)
+        #self.assertEqual(id, ((((1 * 8192) + 2) * 8192) + 3)*8 + 4)
```

Comment:
Where does this fix come from?  I don't think we need to retain the commented out old test, and it would be good if the comment below could be explained more fully, or better, set the variables `tract`, `patchx`, `patchy`, `filter_` and set these in the dataId like this
```
tract = 1
patchx = 2
patchy = 3
filter_ = 4
dataId = dict(tract=tract, patch='{},{}'.format(patchx, patchy), filter='ugrizy'[filter_])
```
and re-write the test like this:
```
self.assertEqual(id, ((((tract * 32) + patchx) * 32) + patchy)*64 + filter_)
```
Then we could remove the commented out lines.

If we could add comments explaining why the values `32`, `32`, and `64` appear, that would be good.